bodhi/server/consumers/signed.py

```diff
@@ -110,10 +111,17 @@ def __call__(self, message: fedora_messaging.api.Message):
             log.info("Build %s has been marked as signed" % build_nvr)
 
             # If every build in update is signed change status to testing
-            if build.update.from_tag and build.update.signed():
+            if not build.update.release.composed_by_bodhi and build.update.signed():
                 log.info("Every build in update is signed, set status to testing")
 
                 build.update.status = UpdateStatus.testing
                 build.update.date_testing = func.current_timestamp()
                 build.update.request = None
+
+                if config.get("test_gating.required"):
```

Comment:
Do you think it's better to have this in `signed.py` instead of `automatic_updates.py`?

Reply:
Since tests are now triggered only when we move the update in testing, I think it makes more sense to set the TestGatingStatus in the same place. Having in in `automatic_updates.py` would work but then we would split the logic related to moving an update to testing in 2 differents consumers.

Reply:
Ok, this makes sense.